src/views/Error404.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import '../globalStyles.css';
+import './Error404.css';
+
+function Error404() {

```

Comment:
```suggestion
function MessageError() {

```
Tal vez podrías cambiar el nombre de este componente.

Reply:
y en la carpeta components